docs/data-sources/equinix_network_device.md

```diff
@@ -49,6 +49,7 @@ NOTE: Exactly one of either `uuid` or `name` must be specified.
   * APPLIED
   * WAITING_FOR_CLUSTER_SETUP
   * REGISTRATION_FAILED
+  * NA
```

Comment:
Documentation for the new field connectivity is also required in both data source and resource.